Actions/Deploy/Deploy.ps1

```diff
@@ -1,6 +1,4 @@
 Param(
-    [Parameter(HelpMessage = "The GitHub token running the action", Mandatory = $false)]
-    [string] $token,
```

Comment:
It's not used in the script but it's passed in the action YAML (Actions/Deploy/action.yaml).

Reply:
You're correct, I missed that. But no workflow overrides the default value and the token still goes unused, so shouldn't it go? Or is there something on the roadmap that would require it?

Reply:
It also needs to be removed from the README.md (in the Actions/Deploy folder)